### PR TITLE
prep-javascript-variables - Marc Ting

### PR DESCRIPTION
**Pull request by mjlting**
Screen Shot 2023-02-27 at 9 24 21 PM

### CODE REVIEW

File: prep-javascript-variables/index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    const fullName = 'Marc Ting';
+    console.log(fullName);
+    console.log(typeof fullName);
+    var numberOfPets = 1;
```
Comment (BrettAlbright) [inline prep-javascript-variables/index.html]: make sure to declare all of the variables using `const` except for the never variable, which should be declared with `let`.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
Looking good Marc, just need to update the variable declarations where you used var

File: prep-javascript-variables/index.html (hunk 1 above)
Comment (mjlting) [reply prep-javascript-variables/index.html]: I made changes to the variables to the variables. Thank you for the feedback!
Screen Shot 2023-02-28 at 5 19 03 PM

Comment (mjlting) [reply prep-javascript-variables/index.html]: Attached is also a picture of the terminal output
Screen Shot 2023-02-28 at 6 17 32 PM

**Review by mjlting — COMMENTED** [review]
My new code output 
Screen Shot 2023-03-01 at 7 22 13 PM

**Review by BrettAlbright — APPROVED** [review]
Great work Marc!

**Pull request merged**